doc/en/writing_plugins.rst

```diff
@@ -454,7 +454,7 @@ hook wrappers and passes the same arguments as to the regular hooks.
 
 At the yield point of the hook wrapper pytest will execute the next hook
 implementations and return their result to the yield point in the form of
-a :py:class:`CallOutcome <_pytest.vendored_packages.pluggy._CallOutcome>` instance which encapsulates a result or
+a :py:class:`CallOutcome <_pytest._pluggy._CallOutcome>` instance which encapsulates a result or
```

Comment:
Could you please create the docs locally (`tox -e docs`) to ensure the changes links still work?

Reply:
@nicoddemus yep I'll test it out

Reply:
seems to have worked 👍